dbld/rules

```diff
@@ -19,6 +19,7 @@ TARBALL_IMAGE ?= tarball
 DEFAULT_DEB_IMAGE=ubuntu-bionic
 DEFAULT_RPM_IMAGE=centos-7
 DOCKER=docker
+DOCKER_IMAGE_PREFIX=balabit
```

Comment:
Unfortunately DockerHub and GitHub namespacing differs a little. GitHub also adds the repository owner to the path. 

Should we default to `ghcr.io/syslog-ng`? It is a little bit strange for me in case of forked repositories, but it would mean the same as `balabit` on DockerHub.

Reply:
IMO the case of the forked repositories is answered the same as #3783. Most probably no one uses this other than us. But if some actually does, they can 1. change the default, 2. set it in rules.conf, 3. set it in env.

So I think it is okay to change the default to `ghcr.io/syslog-ng`, the namespacing difference is not a problem.